lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -577,12 +581,25 @@ PYI_ESCAPE_M2B:
     type: page
     pageId: pyi-escape-m2b
   events:
+    end:
+      targetState: RETURN_TO_RP
     next:
       targetState: IDENTITY_START_PAGE
     bankAccount:
       targetState: BANK_ACCOUNT_START_PAGE
+
+PYI_ESCAPE_ABANDON_M2B:
+  response:
+    type: page
+    context: abandon
+    pageId: pyi-escape-m2b
+  events:
     end:
       targetState: RETURN_TO_RP
+    next:
+      targetState: IDENTITY_START_PAGE
+    bankAccount:
+      targetState: BANK_ACCOUNT_START_PAGE
```

Comment:
If you think the test is enforcing this incorrectly, then we may want to change that instead - I don't think we should be adding routes that don't exist! The assumption here is that a given page id will always need to handle a consistent set of events...

Reply:
Yeah I'm thinking of changing it so we are instead testing if given a page id AND context we handle a consistent set of events